ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -368,6 +368,8 @@ EFVSC:
   - exhaust_fan_speed_percentage_sensor
   - exhaust_fan_current_sensor
   - exhaust_fan_power_sensor
+  - exhaust_fan_run_mode
+  - bypass_exhaust_fan_status
```

Comment:
what is this? bypassing the exhaust fan? why would this be on the vfd control?

Reply:
This only bypasses the drive in case of failure and motor runs at full speed. This field shows whether the motor is running in VFD control or is bypassed.

Reply:
then we would create a field called `exhaust_fan_speed_mode` which would have states of `BYPASS` and `VFD`. see [here](https://github.com/google/digitalbuildings/blob/accdc3b2685b08531cd83fcdd048f432bb2a2008/ontology/yaml/resources/fields/telemetry_fields.yaml#L1513) for an example.

Reply:
Updated